src/components/card.js

```diff
@@ -25,13 +26,25 @@ const Card = ({ id, name, permissions, icon }) => {
   //   setTimeout(fetchInfo, 1500 * index)
   // })
 
+
+  const leaveServer = async (e) => {
+    const resp = await axios.delete(`https://discordapp.com/api/users/@me/guilds/${id}`, 
+        {
+            headers : {
+                Authorization : "Bot " + token
+            }
+        }
+    );
+  }
+
   return (
     <CardWrapper>
       <CardImg src={iconBaseURL} alt={name} />
       <TextWrapper>
         <h3>{name}</h3>
         <p>ID: {id}</p>
         <p>Permissions: {permissions}</p>
+        <button onClick = {leaveServer}> Leave Server </button>
```

Comment:
can you create a styled component and create a simple button style that fits in with the rest? The default one sticks out 😅

Reply:
Yep I'll add some premade button to make it look nice